src/xcode/ENA/ENA/Source/AppDelegate & Globals/NotificationManager.swift

```diff
@@ -44,9 +44,12 @@ final class NotificationManager: NSObject, UNUserNotificationCenterDelegate {
 		}
 	}
 
+	// swiftlint:disable:next cyclomatic_complexity
```

Comment:
Is there really no other way?

Reply:
@marcussc  not really, it is a delegate function and this is a new case, another way is to it to checkForLocalNotificationsActions() function which is called in the default statement of the switch but this function only handles health Certificate notifications so far, what do you think?